app/models/concerns/helpable.rb

```diff
@@ -7,6 +7,27 @@ module Helpable
   included do
     include Bolognese::DoiUtils
     include Cirneco::Utils
+    include Cirneco::Api
+
+    attr_accessor :username, :password
+
+    def register_url
```

Comment:
So its uses cirneco and in place uses MDS, right? @mfenner

Reply:
Yes, cirneco to update the URL in the handle system via MDS API.

Reply:
I haven't tested the full integration, but, together with using basic auth in `bracco`, it solves the issue of talking to the handle server. It also means we can remove a similar call in `cheetoh`, so that `cheetoh` is no longer talking to the MDS API, but only to `lupo`.